### PR TITLE
Autofix Rubocop Batch 1

### PR DESCRIPTION
**Pull request by nschonni**
Removed the commit bit rule that was fixed separately and cleaned a few rules.
I added a dev dependency for rubocop here, but I skipped that in the `ox` repo

### CODE REVIEW

File: oj.gemspec [hunk 1]
```diff
@@ -20,15 +19,16 @@ Gem::Specification.new do |s|
   s.summary = "A fast JSON parser and serializer."
   s.description = "The fastest JSON parser and object serializer."
   s.licenses = ['MIT']
-  s.required_ruby_version = ">= 2.4"
+  s.required_ruby_version = ">= 2.5"
```
Comment (nschonni) [inline oj.gemspec:22]: Changed this to 2.5 to match was is tested in CI, although the version is EOL already

Comment (ohler55) [reply oj.gemspec:22]: Sounds good

**Review by ohler55 — APPROVED** [review]

**Comment by nschonni** [comment]
Aside: do you plan on removing support for the 2.x branches that are EOL? https://endoflife.date/ruby
Even 2.7 drops support next month

**Comment by ohler55** [comment]
Usually the unsupported ones are dropped off the list but I tend to get behind on the updates. I have no problem dropping old ones but maybe keep the ones (2.7) for a bit longer.

**Comment by nschonni** [comment]
Should I do that in here, or a clean PR?

**Comment by ohler55** [comment]
might as well do it here

**Review by ohler55 — APPROVED** [review]

**Comment by nschonni** [comment]
Hmm, not sure about the TruffleRuby failure

**Comment by ohler55** [comment]
Nothing changed that that I can see would impact that test. Odd.

**Comment by nschonni** [comment]
An now TruffleRuby is happy again on this one. I wonder if it's related to the dev_dependency changes. I rebased this out of this one and it's passing, but the other one were I rebase and kept it in is failing still

**Comment by ohler55** [comment]
Can't argue with success.

**Pull request merged**